frontend/src/components/ModalLink.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Button from './Button'
```

Comment:
was this used?

Reply:
Resolved in commit f24c30e8b814ee8bc691b9803db23a6fc53234a6.